portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,13 @@
 
 async function getData() {
     const response = await fetch('/data');
-    const data = await response.text();
-    document.getElementById('data-container').innerHTML = data;
+    const comments = await response.json();
+    let html = "";
+    for(const comment of comments) {
+        html += "<div id=\"comment\">\n";
+        html += "<p>" + comment.message + "</p>\n";
+        html += "<p><small>" + comment.firstName + " " + comment.lastName + "</small></p>\n";
```

Comment:
You might be interested in [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) which would let you do things like

```
html += `<p>${comment.message}</p>`
```

Reply:
Neat! I didn't know about these. I changed both of the lines.